testing/matrix/linter/rules/modules/images.go

```diff
@@ -33,7 +33,8 @@ func skipModuleImageNameIfNeeded(filePath string) bool {
 		// Following images will be removed soon
 		"/deckhouse/modules/110-istio/images/operator-v1x9x1trustca/Dockerfile",
 		"/deckhouse/modules/110-istio/images/pilot-v1x9x1trustca/Dockerfile",
-		"/deckhouse/modules/110-istio/images/proxyv2-v1x9x1trustca/Dockerfile":
+		"/deckhouse/modules/110-istio/images/proxyv2-v1x9x1trustca/Dockerfile",
```

Comment:
I suppose we do not need images bove anymore.

Reply:
yes, thanks !